rhizome/postgres/bin/initialize-empty-database

```diff
@@ -4,4 +4,4 @@
 require_relative "../../common/lib/util"
 
 r "chown postgres /dat"
-r "pg_createcluster 16 main"
+r "LC_ALL=C pg_createcluster 16 main"
```

Comment:
Sorry for not looking into this PR earlier, I was mostly busy with HA recently.

pg_createcluster has --locale option. I think it has exactly same effect with this. [Its docs](https://manpages.ubuntu.com/manpages/trusty/man8/pg_createcluster.8.html) says this.
```
--locale=locale
    Set the default locale for the database cluster. If this option is not specified, the
    locale is inherited from the environment that pg_createcluster runs in.
```